src/App/ApiActions/getRating.js

```diff
@@ -0,0 +1,6 @@
+import axios from '../../configs/axios';
+
+export default async slug => axios
```

Comment:
@alainburindi what's going on here? Why are you not using the error messages gotten from the server?

Reply:
The error message is coming from the backend when the article doesn't have any available statistics which is normal. then I was returning `0` to make sure I'm handling all possible cases. then I was checking if the value is equal to 0 or not. Do you suggest anything else??

Reply:
@alainburindi my actual question is: why do you need this file at all? Why not make the api call as a Redux action and dispatch appropriate messages to the reducer based on responses from the server?

Reply:
While fetching the user's profile I was getting his articles but each article must have his own statistics. So I had to map from the response and get appropriate statistics, push them to the corresponding article and dispatch after